src/Symfony/Component/Cache/Tests/Adapter/SimpleCacheAdapterTest.php

```diff
@@ -26,6 +26,6 @@ class SimpleCacheAdapterTest extends AdapterTestCase
 
     public function createCachePool($defaultLifetime = 0)
     {
-        return new SimpleCacheAdapter(new Psr6Cache(new FilesystemAdapter()), '', $defaultLifetime);
+        return new SimpleCacheAdapter(new FilesystemCache(), '', $defaultLifetime);
```

Comment:
isn't this reverting your recent change ?

Reply:
correct - and the updated test case is now in Psr16AdapterTest